docs(com1DFA, com2AB): clarify optional files and output naming (#1121)
avaframe/AvaFrame: docs/moduleCom1DFA.rst

  - if you want to simulate different scenarios with the same features, you have to copy them to separate shapefiles

> and the following files are optional. Please note: in the standard configuration, the *null* variant is always run! I

Reviewer: ```suggestion
and the following files are optional. Please note: in the standard configuration (i.e. ``simTypeList = available``) , the *null* variant is always run! I
```

> .e. if some of the following files are given, at least two results are generated: the *null* variant and the variant

Reviewer: ```suggestion
.e. if a resistance and/or an entrainment file is given (as described below), at least two results are generated: the *null* variant and the variant
```

> with entrainment and/or resistance.

Reviewer 2: consider adding and secondary release areas are always triggered as soon as any part of the secondary release are is affected by the snow avalanche flow

        This kind of storage of configurations from actually performed simulations allows a run that has been terminated
        to be resumed without re-running simulations that have already been performed. For this, just restart the run.

The naming of the output files has the following structure, shown with the example of
*relAlr_ff5f9b78c6_C_L_null_dfa_ppr*:

* *relAlr* - release area name, usually the name of the shapefile

> * *ff5f9b78c6* - hash of the configuration file used for the simulation. All files related to this simulation

Reviewer: ```suggestion
* *ff5f9b78c6* - individual hash of the configuration file used for the simulation. All files related to this simulation
```

  have the same hash in their name. This allows to identify which files belong to which simulation.
* *C* - indicator of the setup used: D for default setup, C for custom setup, i.e. something was changed in the
  configuration file

> * *L* - indicator of the size catorgory used for the friction model: L for large, M for medium, S for small

Reviewer: ```suggestion
* *L* - indicator of the size category used for the friction model: L for large, M for medium, S for small
```
